libsrc/target/cpm/fcntl/read.c

```diff
@@ -80,7 +80,7 @@ ssize_t read(int fd, void *buf, size_t len)
             fc->rwptr += size;
             len -= size;
         }
-        setuid(uid);
+        setuid(uid);      
```

Comment:
extra whitespace + no change

Reply:
This has already been flowed into [master](https://github.com/z88dk/z88dk/commit/88332e6527717f30e6a1cbade598344b80b77e7d). Just unwinding it out of here.